src/encoder/encodermp3settings.cpp

```diff
@@ -0,0 +1,135 @@
+/**
+* @file encodermp3settings.cpp
+* @author Josep Maria Antolín
+* @date Feb 27 2017
+* @brief storage of setting for mp3 encoder
+*/
+
+#include "encoder/encodermp3settings.h"
+#include "recording/defs_recording.h"
+
+const int EncoderMp3Settings::DEFAULT_BITRATE_INDEX = 6;
+const QString EncoderMp3Settings::ENCODING_MODE_GROUP = "MP3_VBR_MODE";
+
+EncoderMp3Settings::EncoderMp3Settings(UserSettingsPointer pConfig) :
+    m_pConfig(pConfig)
+{
+    // Added "32" because older settings started at index 1 with 48.
+    m_qualList.append(32);
+    m_qualList.append(48);
+    m_qualList.append(64);
+    m_qualList.append(80);
+    m_qualList.append(96);
+    m_qualList.append(112);
+    m_qualList.append(128);
+    m_qualList.append(160);
+    m_qualList.append(192);
+    m_qualList.append(224);
+    m_qualList.append(256);
+    m_qualList.append(320);
+    
```

Comment:
Can we add here a comment which bitrate compares to which V value? 
We have a 12 -> 10 issue here :-/